bsyncviewer/lib/validator/workflow.py

```diff
@@ -165,6 +192,9 @@ def process_use_case(self, use_case):
         return results
 
     def process_errors(self, results):
+        def format_error(error):
+            return f'line {error["line"]}: element {error["element"]}: {error["message"]}'
+
```

Comment:
Happy to change this if a different format is preferred

Reply:
this looks good!